### DISCUSSION TITLE
CMI Nametag

### DISCUSSION THREAD

**Discussion by Shadoves** [Zrips/CMI#3336] [Feature Requests]
Hello !

I would recommend a new feature: Nametags.

After the very cool tab features i think a nametag option is very necessary.( 

Maybe there should be a config where i can customize the groups with custom permissions like. cmi.ntag.<group> if someone has this permission it should have for ex &7(&bB&7) &8<name>

Maybe also a Papi /Mwd function would be nice.

Why this feature? I just know NTE which is free but the most others costs 10$ .

And i think nametags and tab should be always both inside one plugin.

Many thanks to the developers for the good work and the fast fixes.

Best greetings
Shadow

**Comment by mercurialmusic** [comment]
Duplicates #733, #1760, #1778, #2426, & #2600 =p

**Comment by arturek1666** [comment]
You can see dublicates ;) as sooo many people want this feature, what are you waiting for add it now "DO IT"
